dataprofiler/profilers/base_column_profilers.py

```diff
@@ -186,6 +186,22 @@ def _add_helper(self, other1, other2):
                                                    other2.times)
 
         self.sample_size = other1.sample_size + other2.sample_size
+        
+    def diff(self, other_profile, options=None):
+        """
+        Finds the differences for columns.
+
+        :param other_profile: profile to find the difference with
+        :type other_profile: BaseColumnProfiler
+        :return: the stat differences
+        :rtype: dict
+        """
+        cls = self.__class__
+        if not isinstance(other_profile, cls):
+            raise TypeError("Unsupported operand type(s) for diff: '{}' "
+                            "and '{}'".format(cls.__name__,
+                                              other_profile.__class__.__name__))
+        return {}
```

Comment:
technically, all these have `sample_size`, is sample_size part of the diff? if so we can do that here for all

Reply:
I assume the intent of this for the error check, otherwise i'd suggest raise not implemented such that everyone does it

Reply:
honestly, I'm not sure where sample_size is propagated from in the profile yet. Intent is two-fold, one is to reduce duplicate code with the error check, and the other is to prevent the compilers from failing if a column doesnt have diff implemented yet